### DISCUSSION TITLE
taro中使用Taulwindcss框架开发微信小程序，函数组件中无法使用全局的样式，函数组件样式不生效

### DISCUSSION THREAD

**Discussion by 18338625417** [NervJS/taro#14334] [Q & A]
![image](https://github.com/NervJS/taro/assets/109427365/2d2b87a9-f21c-45eb-b8de-29853a41989f)
![image](https://github.com/NervJS/taro/assets/109427365/0031d0e3-c7f1-4b73-96e0-8324205184ef)

**Comment by 18338625417** [comment]
有老哥懂解决方法吗？ 文档中没找到对应的解决方法

**Reply by agileago** [reply]
你这是tabbar吧，tabbar貌似确实是用不了全局的样式， tabbar得自己写了
